hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -443,7 +443,8 @@ public boolean scheduleExists(ScheduleID id) {
 				.setAutoRenewPeriod(Duration.newBuilder().setSeconds(account.getAutoRenewSecs()))
 				.setBalance(account.getBalance())
 				.setExpirationTime(Timestamp.newBuilder().setSeconds(account.getExpiry()))
-				.setContractAccountID(asSolidityAddressHex(id));
+				.setContractAccountID(asSolidityAddressHex(id))
+				.setOwnedNfts(account.getNFTsOwned());
```

Comment:
As a rule of thumb, we are using `nft` instead of `NFT` in terms of representation. Lets keep it consistent and use `getNftsOwned` instead of `getNFTsOwned`. That was pushed from the Protobufs reviews and we should be keeping it consistent across the codebase